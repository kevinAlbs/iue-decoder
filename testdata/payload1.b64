AQAAAAAAAAAAAAAAAAAAAAACwj+3zkv2VM+aTfk60RqhXq6a/77WlLwu/BxXFkL7EppGsju/m8f0x5kBDD3EZTtGALGXlym5jnpZAoSIkswHoA==