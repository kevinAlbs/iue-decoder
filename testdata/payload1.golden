BlobSubtype=1 (FLE1DeterministicEncryptedValue)
KeyUUID=00000000000000000000000000000000
OriginalBsonType=2 (string)
Ciphertext=C23FB7CE4BF654CF9A4DF93AD11AA15EAE9AFFBED694BC2EFC1C571642FB129A46B23BBF9BC7F4C799010C3DC4653B4600B1979729B98E7A5902848892CC07A0
